src/Utils/avatars.js

```diff
@@ -0,0 +1,52 @@
+const host = 'https://skills.susi.ai';
```

Comment:
Please don't hardcode the URL of the system.

Reply:
@mariobehling made it dynamic using `window.location`. Please check.